src/algopy_testing/models/contract.py

```diff
@@ -46,14 +72,46 @@ def _get_local_states(self) -> dict[str, algopy.LocalState[Any]]:
 
         return local_states
 
+    def _get_global_states(self) -> dict[str, algopy.GlobalState[Any]]:
+        import algopy
+
+        global_states = {
+            name: value
+            for name, value in vars(self).items()
+            if isinstance(value, algopy.GlobalState)
```

Comment:
this should also consider other types that don't use the `GlobalState` proxy e.g.

```
self.some_uint = UInt64(0)
self.some_bytes = Bytes(b"")
self.some_asset = Asset(1234) # algopy type is Asset, but is a uint64 in terms of global state storage
self.some_address = Account() # algopy type is Account, but is a bytes in terms of global storage
```